tests/menu/menu.cpp

```diff
@@ -608,3 +608,176 @@ void MenuTestCase::Events()
                           src );
 #endif // wxUSE_UIACTIONSIMULATOR
 }
+
+namespace
+{
+    void verifyAccelAssigned( wxString labelText, int keycode )
```

Comment:
Another nitpick: please follow wx `CamelCase` naming convention:
```suggestion
    void VerifyAccelAssigned( wxString labelText, int keycode )
```

Also, we usually don't indent the namespace contents to avoid having too big indented blocks.